UnitTests/BraintreeTestShared/Encodable+Extension.swift

```diff
@@ -1,6 +1,5 @@
 import Foundation
 
-// TODO: - To be removed once entire SDK is formatting POST bodies using Encodable
```

Comment:
Moved this to our tests so we don't need to completely change all of our testing logic, can consider decoding everything in our tests down the road vs converting to dictionaries as they are today